atomsci/ddm/utils/struct_utils.py

```diff
@@ -381,7 +379,7 @@ def _merge_values(values, strategy='list'):
     elif strategy == 'min':
         val = min(values)
     else:
-        raise Exception('Unknown column merge strategy: %s', columnmerge)
+        raise Exception('Unknown column merge strategy: %s' % (strategy) )
```

Comment:
`columnmerge` isn't defined, but I think `strategy` is the most useful for this message